lib/vmpooler/pool_manager.rb

```diff
@@ -459,69 +458,17 @@ def _check_snapshot_queue
       end
     end
 
-    def migration_limit(migration_limit)
-      # Returns migration_limit setting when enabled
-      return false if migration_limit == 0 || !migration_limit # rubocop:disable Style/NumericPredicate
-      migration_limit if migration_limit >= 1
-    end
-
     def migrate_vm(vm_name, pool_name, provider)
```

Comment:
There is currently no way in Pool Manager to actually add a VM to the migration queue.  So this PR will not have any affect yet.  Is this by design?

Reply:
No, it's an oversight. I'll fix it.